requirements.txt

```diff
@@ -2,6 +2,7 @@ imageio~=2.28.0
 matplotlib~=3.7.0
 networkx~=3.1
 numpy>=1.20,<=1.25
+opencv-python>=4.7.0
```

Comment:
where do we use this dependency?

Reply:
I was previously using it in plot.py to resize the map control graphic when saving the frame to file during gif creation. That plotting function got removed so I'll remove this in the next commit - thanks!

Reply:
Which function was that and why did you remove it?

Reply:
It was the function that created the map control graphic (minimap and line plot); I forget what the function was called exactly. Peter and I decided a little over a week ago to remove the minimap + line plot graphic function since it wasn't a necessity and causing some issues with testing/getting too complicated. Since there are other visualizations available (minimap indivudally and MC metric line plot individually) we decided to remove the MC Graphic from the PR. The code for the function is still in my personal repo, just not included in the branch for this PR.